boilerplate/babel.config.js

```diff
@@ -2,8 +2,6 @@
 const plugins = [
   /** react-native-reanimated web support @see https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation/#web */
   "@babel/plugin-proposal-export-namespace-from",
-  /** NOTE: This must be last in the plugins @see https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/installation/#babel-plugin */
-  "react-native-reanimated/plugin",
```

Comment:
Hi Frank, thanks for your great work on Ignite, as always!
Is the reanimated babel plugin not necessary anymore since Expo 51?
I stilll see it mentioned at the reanimated docs. https://docs.swmansion.com/react-native-reanimated/docs/fundamentals/getting-started/#step-2-add-reanimateds-babel-plugin

Reply:
It's no longer necessary since Expo 50. It's already included in `babel-expo-preset`

https://docs.expo.dev/develop/user-interface/animation/